firefox-ios/Client/Frontend/Autofill/Address/Edit/RemoveAddressButton.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+class RemoveAddressButton: UIButton, ThemeApplicable {
+    private let topSeparator: UIView = {
```

Comment:
```suggestion
    private let topSeparator: UIView = .build { view in
```
Curious if you don't have access to .build as that will allow you to avoid using `translatesAutoresizingMaskIntoConstraints`